### ISSUE TITLE
NodeJS process with WebMIDI crashes upon closing Max 8, if Max was started before starting the NodeJS process

### ISSUE THREAD

**Issue by nnirror** [djipco/webmidi#391]
**Description**

My nodeJS process crashes when I include WebMIDI, under the following circumstance:

I first open Max 8, then start a nodeJS process, and then close Max.

**Environment:**
Specify the environment where you are witnessing the problem:
  - webmidi 3.1.6 for NodeJS
  - NodeJS v14.20.0
  - JavaScript
  - MacOS Catalina

**Details**

Here's the contents of a nodeJS process that's crashing on my machine.  This `test.js` file contains only:

```
const {WebMidi} = require('webmidi');
WebMidi.enable();
```

Steps to reproduce:

1. open Max 8.
2. start the node process: `node test.js`
3. close Max 8.

I see the following error in the console, and I see that the nodeJS process thread has terminated.

Segmentation fault: 11

I checked that the same workflow does not cause a crash with Ableton Live 11.

**Comment by djipco** [comment]
WEBMIDI.js uses the `jzz` package under the hood for MIDI communication on Node.js. Could you install the module and try this basic example to see if you get the same issue: 

```javascript
var navigator = require('jzz');
navigator.requestMIDIAccess().then(onSuccess, onFail);
```

Thanks!

**Comment by nnirror** [comment]
@djipco Yes, I do get the exact same error in a test npm project that only has jzz 1.7.4 installed. When I open Max, start the Node process, and then close Max, I get `Segmentation fault: 11`.
